platforms/substrate/charts/vitalam-ipfs-node/templates/configmap.yaml

```diff
@@ -10,7 +10,7 @@ data:
   {{- if .Values.config.nodeHost }}
   nodeHost: {{ .Values.config.nodeHost }}
   {{- else if .Values.vitalamNode.enabled }}
-  nodeHost: {{ include "vitalam-ipfs-node.fullname" . }}
+  nodeHost: {{ include "vitalam-ipfs.fullname" . }}
```

Comment:
If vitalamNode is enabled, node host will be the name of vitalam node, not ipfs node. It was correct previously.